onnxruntime/python/tools/tensorrt/perf/ort_build_latest.py

```diff
@@ -34,10 +34,12 @@ def install_new_ort_wheel(ort_master_path):
 def main():
     args = parse_arguments()
 
-    p1 = subprocess.Popen(["sudo", "wget", "https://cmake.org/files/v3.17/cmake-3.17.4-Linux-x86_64.tar.gz"])
-    p1.wait()
+    cmake_tar = "cmake-3.17.4-Linux-x86_64.tar.gz" 
+    if not os.path.exists(cmake_tar):
+        p1 = subprocess.Popen(["wget", "-c", "https://cmake.org/files/v3.17/" + cmake_tar])
+        p1.wait()
 
-    p1 = subprocess.Popen(["tar", "zxvf", "cmake-3.17.4-Linux-x86_64.tar.gz"])
+    p1 = subprocess.Popen(["tar", "zxvf", cmake_tar])
```

Comment:
alternatively use python tarfile module.
prefer using python modules when the functionality is available vs using subprocess